docs/common/istio-ambient-update.adoc

```diff
@@ -0,0 +1,653 @@
+default   2           2       1        default-v1-26-4     Healthy   v1.26.4   7d1h
+----
+
+. List the IstioRevision resources to see both versions:
++
+[source,bash,subs="attributes+",name="ambient-revision-based-strategy"]
+----
+$ kubectl get istiorevision -n istio-system
+NAME              TYPE    READY   STATUS    IN USE   VERSION   AGE
+default-v1-26-3   Local   True    Healthy   True     v1.26.3   7d
+default-v1-26-4   Local   True    Healthy   False    v1.26.4   2m
+----
++
+The old revision shows `IN USE: True` because workloads are still connected to it. The new revision shows `IN USE: False` until workloads are migrated.
+
+. Confirm both control plane pods are running:
++
+[source,bash,subs="attributes+",name="ambient-revision-based-strategy"]
+----
+$ kubectl get pods -n istio-system -l app=istiod
+NAME                                      READY   STATUS    RESTARTS   AGE
+istiod-default-v1-26-3-6bd6b8664b-x7k2m   1/1     Running   0          7d
+istiod-default-v1-26-4-7c8e9d775c-y8l3n   1/1     Running   0          2m
+----
+
+. Verify the new control plane is functioning by checking its logs:
++
+[source,bash,subs="attributes+",name="ambient-revision-based-strategy"]
+----
+$ kubectl logs -n istio-system istiod-default-v1-26-4-7c8e9d775c-y8l3n --tail=50 | grep -i "version\|ready"
+----
+
+After creating the new Istio control plane revision, proceed to update IstioCNI and ZTunnel. For IstioCNI update procedures, see link:../../README.md#updating-istiocni[IstioCNI Update Procedure in README]. For ZTunnel update instructions, refer to section "Common Update Procedures for Ambient Components". Note that IstioCNI is compatible with multiple control plane versions and will continue to handle traffic for both the old and new control planes during the migration period.
+
+[[migrate-ambient-workloads-to-new-revision]]
+==== Migrate Ambient Workloads to New Revision
+
+Unlike sidecar mode, ambient mode workloads don't use namespace labels like `istio.io/rev` for version selection. Instead, ambient workloads automatically connect to the active control plane revision. However, to ensure proper migration:
+
+. Verify that your ambient namespaces are still labeled correctly:
++
+[source,bash,subs="attributes+",name="ambient-revision-based-strategy"]
+----
+$ kubectl get namespace bookinfo --show-labels | grep istio
+NAME       STATUS   AGE   LABELS
+bookinfo   Active   7d    istio-discovery=enabled,istio.io/dataplane-mode=ambient
+----
+
+. The ambient workloads automatically use the new control plane. Verify connectivity:
++
+[source,bash,subs="attributes+",name="ambient-revision-based-strategy"]
+----
+$ istioctl ztunnel-config workloads --namespace ztunnel | grep bookinfo
+----
+
+. For more controlled migration, you can temporarily restart application pods to ensure they pick up any configuration changes:
++
+[source,bash,subs="attributes+",name="ambient-revision-based-strategy"]
+----
+$ kubectl rollout restart deployment -n <NAMESPACE>
```

Comment:
directly working in the bookinfo ns.

-n <NAMESPACE> --> bookinfo

kubectl rollout restart deployment -n bookinfo